conseil-lorre/src/main/resources/application.conf

```diff
@@ -37,10 +37,10 @@ lorre {
     block-rights-fetching-is-on: true
     block-rights-fetching-is-on: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
     metadata-fetching-is-on: false
-    metadata-fetching-is-on: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
+    metadata-fetching-is-on: ${?CONSEIL_LORRE_METADATA_FETCHING_ENABLED}
     fork-handling-is-on: false
     fork-handling-is-on: ${?CONSEIL_LORRE_FORK_DETECTION_ENABLED}
-    registered-tokens-is-on: true
+    registered-tokens-is-on: false
```

Comment:
How come we are turning this off by default?

Reply:
I turned it off because of my tests, but now thinking about it if we are publishing it to the public as an external user I would probably want this to be turned off.
I can change it if you wish.